addons/website_event/views/event_event_views.xml

```diff
@@ -8,8 +8,13 @@
         <field name="inherit_id" ref="event.view_event_form"/>
         <field name="arch" type="xml">
             <xpath expr="//group[@name='right_event_details']/field[@name='company_id']" position="after">
+                <field name="website_id" invisible="1"/>
                 <field name="website_id" options="{'no_create': True}" domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]" groups="website.group_multi_website"/>
             </xpath>
+            <xpath expr="//field[@name='tag_ids']" position="attributes">
+                <attribute name="context">{'propagate_website_id': website_id}</attribute>
```

Comment:
Might be worth a comment to say "we use a specific context key to avoid the filtering of `default` context values between modals", either here or in the python default_Get

Reply:
Not sure to understand the comment ^^ why do we need some custom keys ?

Reply:
Right, this was before we settled for storing the field. The idea was that if you create an event tag, and then create a category from there, we should use the website_id of the event we came from.
-> For this you can't use a context key with `default_` because they don't propagate across modals, which makes sense.

Now we can probably just use the website_id in the form view for tags and propagate the website_id of the tag itself actually..

-> Same for the website mixin, since we store it anyway I guess we can just use that